autoware_launch/config/planning/mission_planning/mission_planner/mission_planner.param.yaml

```diff
@@ -9,5 +9,5 @@
     reroute_time_threshold: 10.0
     minimum_reroute_length: 30.0
     consider_no_drivable_lanes: false # This flag is for considering no_drivable_lanes in planning or not.
-    check_footprint_inside_lanes: true
+    check_footprint_inside_lanes: false
```

Comment:
@kosuke55 
Are there any other a lot of false positives?
At least, the case in the description seems easy to be fixed to consider the priority of the orientation to the neighbour lanelets.

Reply:
@takayuki5168 
I haven't analyzed it in detail, but I've seen many false positives so far.
If we fix the case in this video, it might also fix those other cases.
However, lanes are often virtual lines like the video case, so I thought it might be okay to stop checking by default.
The fix of vide case seems easy, but I wasn't keen on allocating the necessary resources.